src/main/kotlin/no/nav/familie/tilbake/dokumentbestilling/manuell/brevmottaker/ManuellBrevmottakerService.kt

```diff
@@ -0,0 +1,80 @@
+package no.nav.familie.tilbake.dokumentbestilling.manuell.brevmottaker
+
+import no.nav.familie.kontrakter.felles.historikkinnslag.Aktør
+import no.nav.familie.tilbake.api.dto.ManuellBrevmottakerRequestDto
+import no.nav.familie.tilbake.common.exceptionhandler.Feil
+import no.nav.familie.tilbake.historikkinnslag.HistorikkService
+import no.nav.familie.tilbake.historikkinnslag.TilbakekrevingHistorikkinnslagstype
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.time.LocalDateTime
+import java.util.UUID
+
+@Service
+class ManuellBrevmottakerService(
+    private val manuellBrevmottakerRepository: ManuellBrevmottakerRepository,
+    private val historikkService: HistorikkService
+) {
+
+    @Transactional
+    fun leggTilBrevmottaker(behandlingId: UUID, manuellBrevmottakerRequestDto: ManuellBrevmottakerRequestDto) {
+        val manuellBrevmottaker = ManuellBrevmottakerMapper.tilDomene(behandlingId, manuellBrevmottakerRequestDto)
+        manuellBrevmottakerRepository.insert(manuellBrevmottaker)
+        historikkService.lagHistorikkinnslag(
+            behandlingId = behandlingId,
+            historikkinnslagstype = TilbakekrevingHistorikkinnslagstype.BREVMOTTAKER_LAGT_TIL,
+            aktør = Aktør.SAKSBEHANDLER,
+            opprettetTidspunkt = LocalDateTime.now()
+        )
+    }
+
+    fun hentBrevmottakere(behandlingId: UUID) = manuellBrevmottakerRepository.findByBehandlingId(behandlingId)
+
+    @Transactional
+    fun oppdaterBrevmottaker(
+        behandlingId: UUID,
+        manuellBrevmottakerId: UUID,
+        manuellBrevmottakerRequestDto: ManuellBrevmottakerRequestDto
+    ) {
+        val manuellBrevmottakere = manuellBrevmottakerRepository.findByBehandlingId(behandlingId)
+        if (manuellBrevmottakere.none { it.id == manuellBrevmottakerId }) {
+            throw Feil("Finnes ikke brevmottakere med id=$manuellBrevmottakerId for behandlingId=$behandlingId")
+        }
```

Comment:
Jeg lurer på egentlig om det er mer gunstig å bare opprette en findByIdAndBehandlingId jdbc repository-metode her som kan returnere nullable ManuellBrevmottaker.

Da slipper man å hente alle manuelle brevmottaker for behandling + å loope mellom dem alle.

Noe ala
```suggestion
        val manuellBrevmottaker = manuellBrevmottakerRepository.findByIdAndBehandlingId(manuellBrevmottakerId, behandlingId) ?:
            throw Feil("Finnes ikke brevmottakere med id=$manuellBrevmottakerId for behandlingId=$behandlingId")
```
På den måten slipper du å ta .single { it.id == manuellBrevmottakerId } på linje 44 også.